db/migrate/20160705085336_rename_template_model.rb

```diff
@@ -1,6 +1,6 @@
 class RenameTemplateModel < ActiveRecord::Migration[5.0]
   def change
-    rename_table :templates, :site_templates
+    rename_table :site_templates, :site_templates
```

Comment:
I think your find and replace might have gone awry here, as renaming a table to the same name makes Postgres unhappy 😀